### PR TITLE
Admin Tweaks

### PR DESCRIPTION
**Pull request by DominikPanic**
Changes:
Adds the "Force Dropship" verb back.
You can now mark adminhelps and if someone marks it before you, it gives you a confirmation warning.
Tiered Tickets
Tiered PMs
Makes a lot of the verbs not shit, code cleanup and fixes
Fixes stat panels
Fixes a few runtimes related to admin stuff

### CODE REVIEW

File: code/game/atoms.dm [hunk 1]
```diff
@@ -246,7 +246,10 @@ its easier to just keep the beam vertical.
 
 /atom/proc/examine(mob/user)
 
-	if(!istype(src, /obj/item) && !istype(src, /obj/effect/statclick))
+	if(istype(src, /obj/effect/statclick))
```
Comment (francinum) [inline code/game/atoms.dm]: Stub this function at the type level.
```
/obj/effect/statclick/examine(mob/user)
	return
```

File: code/game/verbs/ooc.dm [hunk 2]
```diff
@@ -105,7 +105,7 @@ var/global/normal_ooc_colour = "#002eb8"
 	normal_ooc_colour = newColor
 
 /client/verb/round_info()
```
Comment (francinum) [inline code/game/verbs/ooc.dm]: This verb could probably be eliminated entirely, and the `map_tag` added to the Status tab, as on TG.

File: code/modules/admin/fun_verbs.dm [hunk 3]
```diff
@@ -132,21 +132,24 @@
...
-
-	for(var/obj/machinery/computer/communications/C in GLOB.machines)
-		if(!(C.stat & (BROKEN|NOPOWER)))
-			var/obj/item/paper/P = new /obj/item/paper(C.loc)
-			P.name = "'[MAIN_AI_SYSTEM] Update.'"
-			P.info = input
-			P.update_icon()
-			C.messagetitle.Add("[MAIN_AI_SYSTEM] Update")
-			C.messagetext.Add(P.info)
+	if(input("Do you want to use the ship AI to say the message or a global marine announcement?",, "Ship", "Global") == "Ship")
```
Comment (francinum) [inline code/modules/admin/fun_verbs.dm]: This doesn't call properly, and always returns FALSE

File: code/modules/admin/fun_verbs.dm [hunk 4]
```diff
@@ -132,21 +132,24 @@
...
+		ai_system.Announce(input)
+	else
+		command_announcement.Announce(input, MAIN_AI_SYSTEM, new_sound = 'sound/misc/notice2.ogg')
+
+	if(input("Do you want to print out a paper at the communications consoles?",, "Yes", "No") == "Yes")
```
Comment (francinum) [inline code/modules/admin/fun_verbs.dm]: This also fails to call properly.

File: code/modules/admin/fun_verbs.dm (hunk 3 above)
Comment (DominikPanic) [reply code/modules/admin/fun_verbs.dm]: Fixed

File: code/modules/admin/fun_verbs.dm (hunk 4 above)
Comment (DominikPanic) [reply code/modules/admin/fun_verbs.dm]: Fixed

File: code/game/atoms.dm (hunk 1 above)
Comment (DominikPanic) [reply code/game/atoms.dm]: Fixed

File: code/modules/admin/fun_verbs.dm [hunk 5]
```diff
@@ -159,25 +163,26 @@
 	if(!check_rights(R_FUN))
 		return
 
-	var/input = input(usr, "Please enter anything you want. Anything. Serious.", "What?", "") as message|null
-	var/customname = input(usr, "Pick a title for the report.", "Title") as text|null
+	var/input = input("Please enter anything you want. Anything. Serious.", "What?", "") as message|null
```
Comment (Rohesie) [inline code/modules/admin/fun_verbs.dm]:
```suggestion
	var/input = input("Enter what you want announced.", "What?", "") as message|null
```
I'd ask for the title before asking for the main body of the message, though.

**Comment by Rohesie** [comment]
Related: #406 and #409 .

**Pull request merged**